PowerEditor/src/WinControls/Grid/BabyGrid.cpp

```diff
@@ -1725,7 +1730,7 @@ LRESULT CALLBACK GridProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
                        //if there are \n codes in the generic_string, find the longest line
                        longestline=FindLongestLine(hdc,(TCHAR*)lParam,&size);
                        //GetTextExtentPoint32(hdc,(TCHAR*)lParam,lstrlen((TCHAR*)lParam),&size);
-                       required_width = longestline+15;
+					   required_width = longestline+15;

```

Comment:
@eljefe7000 Whitespace changes here and below shouldn't be part of a commit. Was this intended?

Reply:
@chcg my apologies.  It was not intended.  Not sure how that got in there.  My code shows it aligned correctly with the previous line.